### PR TITLE
link frontend to api

### LINKED ISSUE #34: FE - Link seller listing form to API

### PR DESCRIPTION
**Pull request by Justgo13**

### CODE REVIEW

**Review by MillanWang — CHANGES_REQUESTED** [review]
Very good functionality-wise

Could you add some tests that mock an API response from the /listings GET endpoint, and then verify that the info from the response gets shown in the DOM?

File: frontend/src/pages/SellerListings.tsx [hunk 1]
```diff
@@ -1,66 +1,45 @@
...
     const [addListingDialog, setaddListingDialog] = React.useState(false)
     const handleAddListing = () => {
         setaddListingDialog(true)
     }
     const closeDialog = () => setaddListingDialog(false)
+    let { sellerId } = useParams()
+
+    const [sellerItems, setSellerItems] = React.useState([])
+    const { sendRequest } = useHttpClient();
+    React.useEffect(() => {
```
Comment (kfoxIsProgrammer) [inline frontend/src/pages/SellerListings.tsx:26]: This is just a stylistic pet peeve choice. Import React, {useEffect} from "react"

Then just use useEffect as a method itself.

**Comment by Justgo13** [comment]
> Very good functionality-wise
> 
> Could you add some tests that mock an API response from the /listings GET endpoint, and then verify that the info from the response gets shown in the DOM?

added changes

**Review by NosBos — APPROVED** [review]
looks good

**Review by MillanWang — APPROVED** [review]

**Review by kfoxIsProgrammer — COMMENTED** [review]
This looks good

**Pull request merged**